### PR TITLE
unhide svcmap + CHANGELOG

### PR DESCRIPTION
**Pull request by haiut**

### CODE REVIEW

File: cli/config/configStruct.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ type ConfigStruct struct {
 	ConfigFilePath         string                      `yaml:"config-path,omitempty" readonly:""`
 	HeadlessMode           bool                        `yaml:"headless" default:"false"`
 	LogLevelStr            string                      `yaml:"log-level,omitempty" default:"INFO" readonly:""`
-	ServiceMap             bool                        `yaml:"service-map,omitempty" default:"false" readonly:""`
+	ServiceMap             bool                        `yaml:"service-map" default:"false" readonly:""`
```
Comment (IgorGov) [inline cli/config/configStruct.go:41]: To unhide you need to remove the readonly

Comment (IgorGov) [inline cli/config/configStruct.go:41]:
```suggestion
	ServiceMap             bool                        `yaml:"service-map" default:"false"`
```

**Review by RoyUP9 — APPROVED** [review]

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,35 @@
...
+  * Run `mizu check` to verify Mizu was deployed successfully
+  * `mizu check` verifies version compatibility, resources and permissions required by Mizu
+* EXPERIMENTAL: Service Map - graph of all service interactions
+  * Arrow direction show client to server connection
+  * Graph edge width reflects volume of traffic captured between the services
+  * to enable this experimental feature use `--set service-map=true` flag
+
+### improvements
+* Mizu container images are now served from [Docker Hub](https://hub.docker.com/r/up9inc/mizu), as multi-architecture images (arm64, amd64)
+* in Mizu GUI the filter query can now applied by pressing ENTER
```
Comment (RoyUP9) [inline docs/CHANGELOG.md:20]: CONTROL + ENTER

**Review by RoyUP9 — CHANGES_REQUESTED** [review]
small fix

**Review by RoyUP9 — APPROVED** [review]

**Pull request merged**